### DISCUSSION TITLE
Clean up the time tag in the interstitial journal

### DISCUSSION THREAD

**Discussion by kookma** [kookma/TW-Todolist#16] [Q&A]
Question from @applemcg

I've cloned TW-todolist; i'like to clean up the time tag in the interstitial journal. Any pointers would be welcome. I was looking for kookma in TalkTW, but he doesn't seem to have a current user account. i am postponing reaching out on Twitter. Maybe you can help me find him. ~ Thanks, Marty McGowan

See https://github.com/flibbles/tw5-relink/issues/34

**Comment by kookma** [comment]
Data for todolist plugin are stored in system tiddler (those started with `$:`)

For example, https://kookma.github.io/TW-Todolist/#18th%20November%202021 is an interstitial journal
To look at the tiddler contains data for this journal 

1. open $:/ControlPanel
2. open Settings tab
3. open Todolist
4. look for `18th November 2021`
5. you see two tiddlers where you can change everything

NOTE: if you have not setting tab in your wiki (which is visible when you have codemirror plugin install) directly open this tiddler
`$:/plugins/kookma/todolist/settings/internals`

**Reply by applemcg** [reply]
I see the ../data/tasks/18th ... and the ../data/state/18th...

the first time stored in tasks, no doubt a UT 20211118081655926
is clearly 2021, November 18 08:16:55.626

the problem is not in the data, it's the representation of the 0hh:0mm on my view, "editview"?? i see

-----

**03:1** i tried to adopt ...
**6**

-----
the problem must be associated with the conversion and display of the UT to hh:mm?   is there an internal tool  that formats the hh:mm ?

**Reply by kookma** [reply]
See update 1.4.1. The format works now!
